pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -3069,4 +3069,14 @@ self: super: {
     "--extra-include-dirs=${lib.getDev pkgs.opencascade-occt}/include/opencascade"
   ] super.opencascade-hs;
 
+  # https://github.com/haskell-grpc-native/http2-client/pull/96
+  http2-client = overrideCabal (drv: {
+    patches = drv.patches or [ ] ++ [
+      (pkgs.fetchpatch {
+        url = "https://patch-diff.githubusercontent.com/raw/haskell-grpc-native/http2-client/pull/96.patch";
```

Comment:
Please use a patch url that includes the commit hash so the patch is at least somewhat reproducible: <https://github.com/haskell-grpc-native/http2-client/commit/952a57d1138dc31863a5e387dbda67cbdfcfd9f4.patch>

Also, it is good practice to give the patch a meaninful `name`, so it's clear what it does in the build output.

Reply:
thanks, good points, just fixed :+1: